### ISSUE TITLE
`standard-changelog` needs to be updated as well to address trim-newlines (CVE-2021-33623)

### ISSUE THREAD

**Issue by taymoork2** [conventional-changelog/conventional-changelog#878]
`standard-changelog` needs to update `conventional-changelog-writer` to v5 since v5 patched `CVE-2021-33623` which is simple, since the breaking change seems to be internal and not developer facing.
However `meow` needs to be updated as well, which will be a developer facing breaking change.
For `meow` to no longer vulnerable to `CVE-2021-33623`, it needs to be bumped from v8-->v10, which drops support for Node v10, requiring the engine field to be updated to >=12.
The flag/camelcase change seems to internal for the config `meow` uses, as it uses `camelcase-keys` internally to convert `--same-file` to `sameFile` for the config to understand the existing flags being used, and shouldn't be a bigger issue compared to the node requirement

I'm more than happy to introduce the PR and work to getting it in

**Comment by pieterjandesmedt** [comment]
Any progress on this?

**Comment by taymoork2** [comment]
Nope, I was waiting on some from of approval from the maintainer since of the node bump being a breaking change
On my end, we removed `standard-changelog`, since we're already using `standard-version`, and have switched `standard-changelog` to using `npx` instead

**Comment by dangreen** [comment]
Supported standard-changelog/writer releases no longer use the affected dependency chain. Please open a new issue only with a current-version lockfile and reproduction. Closing as obsolete.
